mockwebserver/src/main/kotlin/mockwebserver3/MockResponse.kt

```diff
@@ -362,6 +365,18 @@ class MockResponse : Cloneable {
     webSocketListener = listener
   }
 
+  /**
+   * Configures this response to be served as a response to an HTTP CONNECT request, either for
+   * doing HTTPS through an HTTP proxy, or HTTP/2 prior knowledge through an HTTP proxy.
+   *
+   * When a new connection is received, all in-tunnel responses are served before the connection is
```

Comment:
Why is it assumed there are multiple tunnel requests? 401?

Reply:
407!